server/db/043-deprecate-one-signal.sql

```diff
@@ -0,0 +1,35 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 43;
```

Comment:
Yes, since this one is going to be merged first, it makes sense that you did 43. It just means that you'll need to use 44 in https://github.com/bravetechnologycoop/BraveSensor/pull/259/files#diff-c0c1c20db77f5ac0bcc6331ee111f3d3442088969d94f8403aad07cc86bcf991 . 

Looks like the dev database already has these changes on it. Did you do it manually or roll it back so that it could run this script automatically during deployment?

Reply:
I manually rolled it back! And yes, I plan to use 44 in the other PR :).